samples/bluetooth/mesh/light/src/main.c

```diff
@@ -51,4 +52,8 @@ void main(void)
 	if (err) {
 		printk("Bluetooth init failed (err %d)\n", err);
 	}
+
+	if (IS_ENABLED(CONFIG_BT_MESH_LIGHT_SAMPLE_NRF52_DFU)) {
+		smp_dfu_init();
```

Comment:
Could `SYS_INIT` be used here like in `thingy53.c`?

Reply:
It could, but does not work out of the box. Trying to run it with `k_work` makes the advertising fail. Using `k_work_delayable` however with a seconds delay seems to work. Not sure if this is a little to ad hoc solution. What do you think @PavelVPV ?